build/packer/ansible/agent-aws.yml

```diff
@@ -22,10 +22,36 @@
     - name: Add a repository for NodeJS
       shell: "curl -fsSL https://rpm.nodesource.com/setup_{{ node_version }} | bash -"
 
+    - name: Install Oracle EPEL Repo
+      yum:
+        name: oracle-epel-release-el9
+        state: latest
+
+    - name: Install yum-utils
+      yum:
+        name: yum-utils
+        state: latest
+
+    - name: Add Docker repository
+      command: yum-config-manager --add-repo https://download.docker.com/linux/rhel/docker-ce.repo
+
+    - name: Install Docker packages
+      yum:
+        name:
+          - docker-ce
+          - docker-ce-cli
```

Comment:
Does `docker compose` get installed as well?

Reply:
Yes, there is another task inside this yaml:
```
- name: Install docker-compose
   get_url:
     url: "https://github.com/docker/compose/releases/download/v{{ docker_compose_version }}/docker-compose-linux-{{ ansible_architecture }}"
     dest: /usr/local/bin/docker-compose
     mode: "u+x,g+x,o+x"
```